src/main/java/xbot/common/simulation/SimulationPayloadDistributor.java

```diff
@@ -0,0 +1,35 @@
+package xbot.common.simulation;
+
+import com.google.inject.Inject;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import xbot.common.injection.wpi_factories.DevicePolice;
+
+public class SimulationPayloadDistributor {
+
+    private DevicePolice police;
+
+    @Inject
+    public SimulationPayloadDistributor(DevicePolice police) {
+        this.police = police;
+    }
+
+    public void distributeSimulationPayload(JSONObject allSensorsPayload) {
+        // For each type underneath
+
+        JSONArray allSensorsArray = (JSONArray)allSensorsPayload.get("Sensors");
+        allSensorsArray.forEach(item -> {
+            JSONObject sensor = (JSONObject)item;
+            String id = (String)sensor.get("ID");
+            JSONObject payload = (JSONObject)sensor.get("Payload");
+
+            System.out.println("ID:"+id+", Value:"+payload);
```

Comment:
debug logging?

Reply:
Haha, yes, nice catch! I'll clean it up.